release-notes.md

```diff
@@ -37,6 +37,9 @@
 * `.line.extension:houseNumberIndication.value[x]` ; constraint ; Removed houseNumberIndication for HouseNumberIndication. ; ZIB-2218
 * `.country.extension:countryCode.value[x]` ; terminology ; Changed codes in LandCodelijsten (LandGBACodelijst and LandISOCodelijst). ; ZIB-2218
 
+## Admission (zib-Admission, nl-core-Admission)
+* New profile of zib Admission on StructureDefinition Encounter.
```

Comment:
You can leave out the 'StructureDefinition', or replace it with resource. 

Profile = StructureDefitionion and the StructureDefinition describes the resource type encounter.